server/http/restapi/marketplace.go

```diff
@@ -2,84 +2,13 @@ package restapi
 
 import (
 	"net/http"
-	"sort"
-	"strings"
 
-	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
-
-	"github.com/mattermost/mattermost-server/v5/model"
-	"github.com/pkg/errors"
 )
 
-type MarketplaceApp struct {
-	Installed bool                     `json:"installed"`
-	Labels    []model.MarketplaceLabel `json:"labels,omitempty"`
-	Manifest  apps.Manifest            `json:"manifest"`
-}
-
 func (a *restapi) handleGetMarketplace(w http.ResponseWriter, req *http.Request, actingUserID string) {
 	filter := req.URL.Query().Get("filter")
 
-	apps, err := a.getApps()
-	if err != nil {
-		httputils.WriteInternalServerError(w, err)
-		return
-	}
-
-	// Filter plugins.
-	var result []MarketplaceApp
-	for _, a := range apps {
-		if appMatchesFilter(a.Manifest, filter) {
-			result = append(result, a)
-		}
-	}
-
-	// Sort result alphabetically.
-	sort.SliceStable(result, func(i, j int) bool {
-		return strings.ToLower(result[i].Manifest.DisplayName) <
-			strings.ToLower(result[j].Manifest.DisplayName)
-	})
-
+	result := a.admin.GetListedApps(filter)
```

Comment:
Before this PR the API endpoint did send a list of apps. Now it's an map which causes a crash in the webapp. Would you mind reverting to the old data structure of a list?

Reply:
That was a mistake (sending the map out of the REST API), will do now.

Reply:
@hanzei not 100% sure how to test it, but should be reverted now